packages/docusaurus-plugin-content-docs/src/sidebars.ts

```diff
@@ -255,14 +256,23 @@ export const DefaultSidebars: UnprocessedSidebars = {
 };
 
 // TODO refactor: make async
-export function loadSidebars(sidebarFilePath: string): UnprocessedSidebars {
-  if (!sidebarFilePath) {
-    throw new Error(`sidebarFilePath not provided: ${sidebarFilePath}`);
+export function loadSidebars(
+  sidebarFilePath: string | false | undefined,
+): UnprocessedSidebars {
+  // In case false value, we have to set to DefaultSidebars first to generate navigation link
+  // See https://github.com/facebook/docusaurus/pull/4775
+  if (typeof sidebarFilePath !== 'string') {
+    return DefaultSidebars;
   }
 
-  // No sidebars file: by default we use the file-system structure to generate the sidebar
-  // See https://github.com/facebook/docusaurus/pull/4582
   if (!fs.existsSync(sidebarFilePath)) {
+    console.log(
```

Comment:
going to move this warning (and make it an error), to its original place, as we should throw in case of file not found, but not for the versioned docs (as a missing file is a possibility). Unfortunately in this function we don't have the current version name so it's not the best place to handle the error.